alaoweb/2020.alaoweb.org
# Adds OLSSI Logo to Additional Support Tier

This PR:
- Adds OLSSI logo to site
- Includes OLSSI in the Additional Support tier for the conference

To Test:
1. Scroll down to conference sponsors
2. Ensure OLSSI logo is in the additional support row
3. Confirm OLSSI logo links to the institute website

Closes #191
